bin/p2-preparer/main.go

```diff
@@ -124,3 +167,12 @@ func waitForTermination(logger logging.Logger, quitMainUpdate chan struct{}, qui
 	quitMainUpdate <- struct{}{}
 	<-quitMainUpdate // acknowledgement
 }
+
+func loadTokens(path string) ([]string, error) {
```

Comment:
nit: would it make more organizational sense to put this kind of generic function in `pkg/util`?